torch/csrc/api/include/torch/autograd.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <torch/csrc/autograd/custom_function.h>
+
+namespace torch {
+  using Variable = autograd::Variable;
```

Comment:
Why do these need to be exposed via torch:: now and not before? they look like autograd stuff no?

Reply:
cc'ing @yf225, IIRC, the desired terminal end state is that we get rid of `Variable` entirely, is that right?

Reply:
Yes that's correct. We should continue using `torch::autograd::Variable` before the terminal end state happens.